go/vt/vtctl/vtctl.go

```diff
@@ -438,6 +438,11 @@ var commands = []commandGroup{
 				method: commandSetKeyspaceServedFrom,
 				params: "[-source=<source keyspace name>] [-remove] [-cells=c1,c2,...] <keyspace name> <tablet type>",
 				help:   "Changes the ServedFromMap manually. This command is intended for emergency fixes. This field is automatically set when you call the *MigrateServedFrom* command. This command does not rebuild the serving graph.",
+			}, {
+				name:   "SetKeyspaceDurabilityPolicy",
+				method: commandSetKeyspaceDurabilityPolicy,
+				params: "[-durability_policy=policy_name] <keyspace name>",
+				help:   "Changes the durability_policy used by the keyspace specified.",
```

Comment:
Do we _have_ to add this to vtctl? Can we do it just with vtctldclient, since it is a new command?
@ajm188 thoughts?

Reply:
++ please just move it to vtctldclient (and grpcvtctldserver) ❤️

Reply:
So I just remove this vtctl part?